vowpalwabbit/cb_adf.cc

```diff
@@ -85,19 +83,22 @@ struct cb_adf
   const bool _rank_all;
   const float _clip_p;
 
+  VW::io::logger _logger;
```

Comment:
nit: inconsistent naming in reductions? `_logger` vs `logger`?

Reply:
I opted to be consistent within this reduction, but it does mean it is globally inconsistent now